tests/library/Respect/Validation/Rules/HexRgbColorTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Respect\Validation\Rules;
+
+class HexRgbColorTest extends \PHPUnit_Framework_TestCase
+{
+    public function testHexRgbColorValuesONLYShouldReturnTrue()
+    {
+        $validator = new HexRgbColor();
+        $this->assertTrue($validator->assert('#0'));
+        $this->assertTrue($validator->assert('#123456'));
+        $this->assertTrue($validator->assert('#1234'));
+        $this->assertTrue($validator->assert('#1234'));
+        $this->assertTrue($validator->assert('#FFFFF'));
+        $this->assertTrue($validator->assert('123123'));
+        $this->assertTrue($validator->assert('FFFFF'));
+        $this->assertTrue($validator->assert(1));
+        $this->assertTrue($validator->assert(0));
```

Comment:
@DavidePastore, to count the number in the hex color, you should use `strlen()`.